roles/webserver/templates/nginx_unicorn.j2

```diff
@@ -18,7 +18,7 @@ server {
 
   include {{ app }}_ssl;
 
-  server_name www.{{ domain }};
+  server_name {{sub_domain}} {{ domain }};
```

Comment:
Actually I've noticed the `domain` and `protocol` vars come from the `mortik.nginx-rails` role. So cannot change them just like that. I think `mortik.nginx-rails` assumes a bare domain so am wondering about changing the default nginx config.